epytope/Core/Peptide.py

```diff
@@ -85,6 +85,22 @@ def __repr__(self):
             lines.append("in PROTEIN: %s" % p_id)
         return '\n'.join(lines)
 
+    """
+    Determines whether the :class:`~epytope.Core.Peptide.Peptide` is generated from a variant in the corresponding :class:`~epytope.Core.Protein.Protein`.
+
+    :return: True if peptide originates from a variant
+    :rtype: boolean
+    """
+    def is_created_by_variant(peptide):
```

Comment:
Shouldn't it be `Self` instead of `peptide`?

Reply:
Yes 👍, classic copy-paste error...